### DISCUSSION TITLE
gmt should send --help output to STDOUT, not STDERR

### DISCUSSION THREAD

**Discussion by jidanni** [GenericMappingTools/gmt#7616] [Documentation development discussion]
```
$ cat  --help | wc
     26     139    1178
$ pr   --help | wc
     80     471    4233
$ diff --help | wc
    100     651    5388
$ gmt  --help | wc
      0       0       0
$ gmt --help 2>&1 | wc
    190    1419   11414

```
gmt should send --help etc. output to STDOUT, not STDERR.

Else the user's $PAGER can't read it, without first Googling for hacks, like "2>&1".

**Comment by joa-quim** [comment]
I am going to say this once more.

**Please don't open mico-issues**. Discuss things in the forum first if you like.

**Reply by jidanni** [reply]
Sorry. I mistook Discussions for the Forum.

**Comment by PaulWessel** [comment]
Agree with @joa-quim, these are not errors but how we chose to do stuff.  You can argue against these on the forum but do not open issues, please.

**Comment by joa-quim** [comment]
OK, I think we missed that this are not issues but _discussions_, sorry. The interface is so similar that I missed it. It's right that we have those "Discussions" tab but our habit have been to discuss things in the forum.

**Comment by PaulWessel** [comment]
The GitHub discussions is only for developers
